packages/embed-jsx/package.json

```
@@ -0,0 +1,42 @@
+{
+	"name": "@sapphire/embed-jsx",
+	"version": "1.0.0",
```

```suggestion
	"version": "0.0.1",
```

Lerna versioning stupidity but this is needed